subscriptions/models.py

```diff
@@ -9,9 +11,39 @@
 from kukkuu.consts import OCCURRENCE_IS_NOT_FULL_ERROR
 from subscriptions.consts import NotificationType
 
+if TYPE_CHECKING:
+    from users.models import User
+
 
 class FreeSpotNotificationSubscriptionQueryset(models.QuerySet):
```

Comment:
Inconsistent wording of query set i.e. `Queryset` vs. `QuerySet`. Using `FreeSpotNotificationSubscriptionQuerySet` would be consistent with Django.